airbyte-integrations/connectors/source-sentry/source_sentry/streams.py

```diff
@@ -26,10 +25,6 @@ def __init__(self, hostname: str, **kwargs):
     def url_base(self) -> str:
         return self._url_base
 
-    @property
-    def availability_strategy(self) -> Optional["AvailabilityStrategy"]:
```

Comment:
As I understand from oncall comments it needs `availability_strategy` to be turned on by default for this connector. But if to just remove the property from the stream, then nothing to be called by default. It must be specified explicitly for something to happen

Reply:
The base class defines this property, so I just removed the override

Reply:
Right, they add it to the base class yesterday